storybook/storybook-react/src/components/UnorderedList/UnorderedList.docs.mdx

```diff
@@ -1,11 +1,38 @@
-import { Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import { Canvas, Meta } from "@storybook/blocks";
 import * as UnorderedListStories from "./UnorderedList.stories.tsx";
-import README from "../../../../../packages/css/src/unordered-list/README.md?raw";
 
 <Meta of={UnorderedListStories} />
 
-<Markdown>{README}</Markdown>
+# Unordered list
 
-<Primary />
+Een lijst is een verticale groep gerelateerde inhoud.
 
-<Controls />
+## Voorbeelden
+
+### Basis
+
+Een ongeordende lijst heeft vierkante opsommingstekens.
+Tussen de items staat extra witruimte.
+Dat geeft een beter onderscheid tussen de items, zeker als ze uit meerdere regels tekst bestaan.
+
+<Canvas of={UnorderedListStories.Basis} />
```

Comment:
The controls are missing here

Reply:
I think we should offer controls on the individual story pages only, to keep the docs page easier to read. That’s also why I’m initially hiding the code now.